packages/react/__tests__/get-label-from-stack-trace.js

```diff
@@ -0,0 +1,560 @@
+  })
+
+  test('SSR', () => {
+    const stackTrace = `Error
+    at Object.createEmotionProps (webpack-internal:///../../packages/react/dist/emotion-element-7a9c77b4.cjs.dev.js:175:40)
+    at jsxDEV (webpack-internal:///../../packages/react/jsx-dev-runtime/dist/emotion-react-jsx-dev-runtime.cjs.dev.js:22:75)
+    at MyComponent$9 (webpack-internal:///./pages/index.js:60:85)
+    at processChild (C:/Projects/OSS/emotion/node_modules/react-dom/cjs/react-dom-server.node.development.js:3043:14)
+    at resolve (C:/Projects/OSS/emotion/node_modules/react-dom/cjs/react-dom-server.node.development.js:2960:5)
+    at ReactDOMServerRenderer.render (C:/Projects/OSS/emotion/node_modules/react-dom/cjs/react-dom-server.node.development.js:3435:22)
+    at ReactDOMServerRenderer.read (C:/Projects/OSS/emotion/node_modules/react-dom/cjs/react-dom-server.node.development.js:3373:29)
+    at Object.renderToString (C:/Projects/OSS/emotion/node_modules/react-dom/cjs/react-dom-server.node.development.js:3988:27)
+    at Object.renderPage (C:/Projects/OSS/emotion/node_modules/next/dist/server/render.js:621:45)
+    at Object.defaultGetInitialProps (C:/Projects/OSS/emotion/node_modules/next/dist/server/render.js:301:51)`
+
+    expect(getLabelFromStackTrace(stackTrace)).toBe(expectedLabel)
+  })
+})
+
+/**
+ * E.g.
+ *
+ * ```
+ * function MyComponent$9() {
+ *   function renderSpan() {
+ *     return <span css={{ color: 'orchid' }}>Orchid</span>
+ *   }
+ *
+ *   return <div>{renderSpan()}</div>
+ * }
+ * ```
+ */
+describe('render function within function component', () => {
+  test('Chrome', () => {
+    const stackTrace = `Error
+    at createEmotionProps (emotion-element-895e3bbe.browser.esm.js:142)
+    at jsxDEV (emotion-react-jsx-dev-runtime.browser.esm.js:18)
+    at renderSpan (App.js:5)
+    at MyComponent$9 (App.js:8)
+    at renderWithHooks (react-dom.development.js:14803)
+    at mountIndeterminateComponent (react-dom.development.js:17482)
+    at beginWork (react-dom.development.js:18596)`
+
+    expect(getLabelFromStackTrace(stackTrace)).toBe(expectedLabel)
+  })
+
+  test('Firefox', () => {
+    const stackTrace = `createEmotionProps@http://localhost:3000/static/js/main.chunk.js:844:40
+jsxDEV@http://localhost:3000/static/js/main.chunk.js:1126:247
+renderSpan@http://localhost:3000/static/js/main.chunk.js:2273:89
+MyComponent$9@http://localhost:3000/static/js/main.chunk.js:2287:15
+renderWithHooks@http://localhost:3000/static/js/vendors~main.chunk.js:16616:31
+mountIndeterminateComponent@http://localhost:3000/static/js/vendors~main.chunk.js:19228:17
+beginWork@http://localhost:3000/static/js/vendors~main.chunk.js:20306:20
+beginWork$1@http://localhost:3000/static/js/vendors~main.chunk.js:24848:18`
+
+    expect(getLabelFromStackTrace(stackTrace)).toBe(expectedLabel)
+  })
+
+  test('Safari', () => {
+    const stackTrace = `createEmotionProps@http://localhost:3000/static/js/main.chunk.js:844:49
```

Comment:
that's a funny case :P since in here the TCOed function is not relevant for us, it still works in the Safari - which is kinda of no surprise